apps/backend/src/reconciliation/pos-reconciliation.service.ts

```diff
@@ -376,4 +522,63 @@ export class PosReconciliationService {
       paymentExceptions: paymentExceptions.length,
     };
   }
+  /* eslint-disable @typescript-eslint/no-explicit-any */
+  /* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
We ideally don't want this, and definitely don't want this twice

Reply:
haha, I will remove the one... I am using any on the acc, but still returning the proper types.

Reply:
My argument is that acc can be defined!

Reply:
Oh, oops. Will fix.